pyswarms/utils/search/base_search.py

```diff
@@ -33,6 +33,8 @@ def __init__(
         options,
         objective_func,
         iters,
+        obj_args=(),
+        obj_kwargs={},
```

Comment:
Is it possible to avoid `obj_args=()` and `obj_kwargs={}`? If not, just revert this to the original. I'm planning to update its API soon